### PR TITLE
adding commands for mutating guppy index

### PR DESCRIPTION
**Pull request by atharvar28**

### CODE REVIEW

**Review by jingh8 — APPROVED** [review]

File: suites/portal/studyViewerTest.js [hunk 1]
```diff
@@ -5,15 +5,44 @@ const { Bash } = require('../../utils/bash.js');
...
+        const errMsg = `${new Date()}: The new guppy pod never came up with the expected indices: Details: ${guppyStatusCheckResp.data}`;
+        console.log(errMsg);
+        console.log(`err: ${guppyStatusCheckResp.data}`);
+        throw new Error(`ERROR: ${errMsg}`);
+      }
+    }
+  }
+  // wait a bit for any old pod replicas to go away
+  await sleepMS(30000);
+  // Other arborist policy revoking stuff
```
Comment (themarcelor) [inline suites/portal/studyViewerTest.js:44]: This comment would be more suitable in the AfterSuite block.

File: suites/portal/studyViewerTest.js [hunk 2]
```diff
@@ -5,15 +5,44 @@ const { Bash } = require('../../utils/bash.js');
 const studyViewerTasks = require('../../services/portal/studyViewer/studyViewerTasks.js');
```
Comment (themarcelor) [inline suites/portal/studyViewerTest.js:5]: Please add some code comments to help others understand the pre-requisites around this test.

**Review by themarcelor — CHANGES_REQUESTED** [review]
Requesting code comments.

**Review by themarcelor — APPROVED** [review]
+1

**Pull request merged**